sdk/src/main/java/io/radar/sdk/model/RadarPlace.kt

```diff
@@ -118,12 +118,16 @@ class RadarPlace(
     }
 
     fun toJson(): JSONObject {
+        val categoriesArr = JSONArray()
+        categories.forEach { category -> categoriesArr.put(category) }
```

Comment:
Need to explicitly construct a `JSON` array or you end up with `"[java.lang.String]"` instead of an array